.github/workflows/self_hosted_runner_utils/setup_runner.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
```

Comment:
README file for all this might be nice. I guess you've got an internal google doc. Maybe link it from these files?